Winget-AutoUpdate/functions/Update-App.ps1

```diff
@@ -18,37 +18,43 @@ Function Update-App ($app) {
     
     #Run Winget Upgrade command
     & $Winget upgrade --id $($app.Id) --all --accept-package-agreements --accept-source-agreements -h | Tee-Object -file $LogFile -Append
-    
+
+    #Set mods to apply as an upgrade
+    $ModsMode = "Upgrade"
```

Comment:
It says "not used" but I think it's a special case https://docs.microsoft.com/sv-se/powershell/utility-modules/psscriptanalyzer/rules/UseDeclaredVarsMoreThanAssignments?view=ps-modules
Because $ModsMode is not used within the scriptblock where it was defined.

Reply:
It is resolved now

Reply:
seems good :)